index.html

```diff
@@ -48,6 +48,12 @@ <h1 class="pl-heading pl-element-title">{{ post.title }}</h1>
                         <h2 class="pl-heading pl-tab-heading pl-element-preview-title">{{post.title}} Preview</h2>
 
                         <div class="pl-tab-content">
+                            {% if post.tags contains "icons" %}
+                                <label for="iconFontSlider" class="is-hidden">Drag to resize icons</label>
+                                <input id="iconFontSlider" type="range" min="12" max="144" step="12" value="24" role="slider" aria-min="12" aria-max="144" aria-now="24">
```

Comment:
NIT: ID name doesn't match our naming conventions. Please sync up id and for attributes with an updated value.